packages/blocks/src/__internal__/service/json2block.ts

```diff
@@ -141,7 +145,10 @@ export async function json2block(
         length: 0,
       });
     } else {
-      // TODO: set embed block selection
+      requestAnimationFrame(() => {
+        // TODO: wait block ready
+        focusBlockByModel(lastModel);
+      });
```

Comment:
What does this mean?

Reply:
Why do you want to focus an image?

Reply:
You can find the context from #3639 

Most document tools, such as Notion and Pages, will focus on the image after pasting, allowing users to easily move, resize, delete and perform other operations on the image. 

While focusing on the image, the system is also able to respond appropriately to multi-paste events.

Reply:
Unfortunately, the selection manager module is currently undergoing a process of refactoring.
This part is temporarily reverted due to a compatibility issue. We will add it back after the refactoring is done.